com.microsoft.java.debug.plugin/plugin.xml

```diff
@@ -7,6 +7,10 @@
             <command id="vscode.java.resolveClasspath"/>
             <command id="vscode.java.resolveMainClass"/>
             <command id="vscode.java.buildWorkspace"/>
```

Comment:
IMO, remote debug session should make changes on the launch.json configuration, but not add another set of debug commands.